README.md

```diff
@@ -15,19 +15,48 @@ This [awesome](https://github.com/sindresorhus/awesome) repository is maintained
+- [Getting Started With InnerSource](https://innersourcecommons.org/learn/books/getting-started-with-innersource/) - Author Andy Oram takes you inside InnerSource, first by reviewing the principles that make open source development successful, and then by describing how InnerSource has worked at PayPal.
+- [Understanding the InnerSource Checklist](https://innersourcecommons.org/learn/books/understanding-the-innersource-checklist/) - A checklist guides you through various steps needed to produce software using the InnerSource approach, whether it’s company wide or just for one team.
+- [Managing InnerSource Projects](https://innersourcecommons.org/learn/books/managing-innersource-projects/) - In this book Daniel Izquierdo & José Manrique López explain basic infrastructure as well as metrics that are helpful when introducing InnerSource methodologies into an enterprise.
+- [Adopting InnerSource](https://innersourcecommons.org/learn/books/adopting-innersource-principles-and-case-studies/) - The case studies candidly discuss the difficulties of getting InnerSource projects started, along with the progress so far and the benefits or negative fallout.
+- [InnerSource Patterns](https://innersourcecommons.org/learn/books/innersource-patterns/) -  This book contains the most mature patterns, collected by the InnerSource Commons Community.
+
 ## Blogs
 
-## Articles 
+- [Sebastian Spier - Blog](https://spier.hu/) - Sebastian Spier is the Board member of the [InnerSource Commons Foundation](https://innersourcecommons.org/).
+
+## Tools and Applications
+
+- [Backstage](https://backstage.io/) - An open platform for building developer portals. Realization of the [InnerSource Portal Pattern](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/innersource-portal.md).
+
+## Articles and Videos 
 
-## Videos
+### About InnerSource
 
-## Repositories
+- [InnerSource Commons](https://innersourcecommons.org/) - InnerSource Commons is the world's largest community of InnerSource practitioners. It is dedicated to creating and sharing knowledge about InnerSource, the use of open source best practices for software development within the confines of an organization.
+
+### Culture
+
+### DevOps
+
+### Metrics, Incentives, ROI, KPI
+- [Knights and Gold Stars: A Tale of InnerSource Incentivization](https://ieeexplore.ieee.org/document/9904025)
+- [Starting the InnerSource Journey: Key Goals and Metrics to Measure Collaboration](https://conf.researchr.org/details/msr-2022/msr-2022-industry-track/12/Starting-the-InnerSource-Journey-Key-Goals-and-Metrics-to-Measure-Collaboration)
+- [Sustaining arm’s length cost allocations for highly integrated development functions – an explorative case study of transfer pricing for InnerSource communities](https://mnetax.com/sustaining-arms-length-cost-allocations-for-highly-integrated-development-functions-an-explorative-case-study-of-transfer-pricing-for-innersource-communities-47288)
 
 ## Adopters
```

Comment:
Rather than listing these individually, I would rather link to https://innersourcecommons.org/stories/, where we also have references pointing to posts/videos/etc where the adopters have shared something about their InnerSource approach.

Reply:
Good idea! I've forgotten this page.